src/components/RightSidebar/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -343,18 +343,40 @@ export default {
 						},
 						password: {
 							type: 'password',
-							placeholder: t('spreed', 'Password'),
+							placeholder: t('spreed', 'Connection password'),
 							icon: 'icon-category-auth',
 						},
 						channel: {
 							type: 'text',
 							placeholder: t('spreed', 'IRC channel'),
 							icon: 'icon-group',
 						},
+						nickservnick: {
+							type: 'text',
+							placeholder: t('spreed', 'NickServ nickname'),
```

Comment:
Looking forward to translations for `NickServ` but also don't have a better idea how to make it clearer for translators

Reply:
Yeah it's most likely going to be translated as "NickServ" as it's a proper name.